Kudu.Core/Deployment/StructuredTextLogger.cs

```diff
@@ -0,0 +1,131 @@
+﻿using Kudu.Core.Tracing;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Kudu.Core.Deployment
+{
+    public class StructuredTextLogger : IDetailedLogger
+    {
+        private readonly static object DocumentLock = new object();
```

Comment:
by rename the file, you mean the `StructuredTextLogger.cs` right? I mean it's in the `Kudu.Core.Deployment` namespace :)
I can change the name, I just think that having to manage a static collection for the locks might be too much for this logger at this point